src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -231,7 +231,13 @@ class Telemetry : public PluginBase {
         LAND, /**< @brief Landing. */
         OFFBOARD, /**< @brief Offboard mode. */
         FOLLOW_ME, /**< @brief FollowMe mode. */
-        UNKNOWN /**< @brief Mode not known. */
+        UNKNOWN, /**< @brief Mode not known. */
```

Comment:
Wait... `UNKNOWN` must be the first one, as per [the proto files](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L238-L246) :thinking:. They must be in the same order in the proto file and in this file. Would you mind moving `UNKNOWN` at the top here, and make sure the others are in the right order?

Reply:
Sure but previous version wasn't like that. UNKNOWN is on the top, Copy that :). 

For others, should I sort like here?

https://github.com/mavlink/MAVSDK/blob/6665fee1ef803188bc7c7552174dfca43460919e/src/core/px4_custom_mode.h#L50-L70

Should they like this?
```
        UNKNOWN, /**< @brief Mode not known. */
        READY, /**< @brief Armed and ready to take off. */
        TAKEOFF, /**< @brief Taking off. */
        HOLD, /**< @brief Hold mode (hovering in place (or circling for fixed-wing vehicles). */
        MISSION, /**< @brief Mission mode. */
        RETURN_TO_LAUNCH, /**< @brief Returning to launch position (then landing). */
        LAND, /**< @brief Landing. */
        OFFBOARD, /**< @brief Offboard mode. */
        FOLLOW_ME, /**< @brief FollowMe mode. */
        MANUAL, /**< @brief Manual mode. */
        ALTCTL, /**< @brief Altitude mode. */
        POSCTL, /**< @brief Position mode. */
        ACRO, /**< @brief Acro mode. */
        STABILIZED, /**< @brief Stabilize mode. */
        RATTITUDE /**< @brief Rattitude mode. */
```

Reply:
They should be in the same order as the proto file (linked above). The reason for that is that we generate code from those proto files. In the future, the C++ headers will be auto-generated, too. But in the meantime we have to take care of that manually =)

Reply:
> Should they like this?

That's correct! Please update that here, and then it's good to be merged :+1: :rocket: